app/components/threads_button/threads_button.test.tsx

```diff
@@ -51,8 +48,6 @@ describe('Thread item in the channel list', () => {
         const {toJSON} = renderWithIntlAndTheme(
```

Comment:
Due to the changes, this test is the same as `Threads Component should match snapshot`, so can be removed.